shared/Events.h

```diff
@@ -385,6 +388,9 @@ extern CdeclEvent <AddressListMulti<0x5B8830, GAME_10EN, H_CALL,
 extern CdeclEvent <AddressListMulti<0x48E49B, GAME_10EN, H_CALL,
                                     0x48E55B, GAME_11EN, H_CALL,
                                     0x48E4EB, GAME_STEAM, H_CALL>, PRIORITY_AFTER,  ArgPickNone, void()> gameProcessEvent;
+extern CdeclEvent <AddressListMulti<0x582E6C, GAME_10EN, H_CALL,
+                                    0x5831AC, GAME_11EN, H_CALL,
+                                    0x000000, GAME_STEAM, H_CALL>, PRIORITY_AFTER, ArgPickNone, void()> initGameEvent;
```

Comment:
If anybody has the address of `initGameEvent` for GTA III steam version, then feel free to edit this.